scss/_navbar.scss

```diff
@@ -289,13 +289,13 @@
           }
 
           .nav-link {
-            padding-right: var(--#{$variable-prefix}navbar-nav-link-padding-x);
-            padding-left: var(--#{$variable-prefix}navbar-nav-link-padding-x);
+            padding-right: var(--#{$prefix}navbar-nav-link-padding-x);
+            padding-left: var(--#{$prefix}navbar-nav-link-padding-x);
 
             // Boosted mod
             &.active::before {
               right: $nav-link-padding-y;
-              bottom: -$navbar-padding-bottom;
+              bottom: $navbar-padding-bottom * -1;
```

Comment:
Reverted in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1167/commits/3e724e8a0e71950f1e4febf33d441354f2909641. I'll modify it another time; it will be easier for you to rebase the Global Headers PR :innocent: